tests/test_runner/test_checkpoint.py

```diff
@@ -146,8 +146,24 @@ def test_get_state_dict():
                         wrapped_model.module.conv.module.bias)
 
 
-def test_load_pavimodel_dist():
+def test_get_torchvision_models():
```

Comment:
Need to use `get_torchvision_models` to test.

Reply:
I applied it from 718ce30